mods/ctf/ctf_classes/medic.lua

```diff
@@ -91,7 +106,11 @@ minetest.override_item("ctf_bandages:bandage", {
 
 				if main and match then
 					local reward = 3
+
 					if ctf_flag.has_flag(pname) then reward = 6 end
+
+					if dont_heal(pname) then reward = 0 end
```

Comment:
This shall fix the error above.
```suggestion
					if dont_heal[pname] then reward = 0 end
```

Reply:
:thumbsup:

Reply:
dang